product_docs/docs/efm/4/05_using_efm.mdx

```diff
@@ -105,7 +105,7 @@ In the event of a failover, Failover Manager first retrieves information from Po
 
     The promotion priority for nodes changes when a new primary is promoted.
 
-    If the `efm set-priority` command was used to change whether a standby is promotable, it may be reset to the value in the standby's properties file through promotion or cluster splits and rejoins.
+    If the `efm set-priority` command was used to change whether a standby is promotable, it may be reset to the value in the standby's properties file through promotion or cluster splits and rejoins. If the agent is restarted, the promotable status will revert to the value in the properties file.
```

Comment:
```suggestion
    If the `efm set-priority` command was used to change whether a standby is promotable, it may be reset to the value in the standby's properties file through promotion or cluster splits and rejoins. If the agent is restarted, the promotable status reverts to the value in the properties file.
```